### PR TITLE
Multiple pipelines API

### PR DESCRIPTION
**Pull request by vladimir-konnov**
Updated swagger docs because of the changes introduced with multiple pipelines feature
P.S. Please leave a comment and set 'Reviewed by' field here http://youtrack.xap.ie/issue/CRM-15217 after review

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -280,6 +280,9 @@ tags:
   description: >-
     A simple, effective way to be notified when things happen in OnePageCRM. For more information, go to the
     'WebHooks & More' tab at the top of the page.
+- name: Pipelines
```
Comment (kevGalway) [inline swagger.yaml]: Trying to keep a very rough order of importance in heading/tags here.This prevents lots of scrolling when using these docs. e.g: webhooks and countries endpoint are at bottom and bootstrap + contact + users are at the top. Was just a rough idea of order that api users would need things. 
Maybe pipelines should be somewhere around deals/attachments?

Comment (kevGalway) [reply swagger.yaml]: Can we add another line about what are pipelines to the description?

Comment (kevGalway) [reply swagger.yaml]: also are pipelines only editable by admins. If so can we add `Only editable by admins.`

File: swagger.yaml [hunk 2]
```diff
@@ -7829,7 +7927,15 @@ components:
           - AU
           - DE
         deal_stages:
-          $ref: '#/components/schemas/Deal_stage'
```
Comment (kevGalway) [inline swagger.yaml]: Nice catch!

**Comment by vladimir-konnov** [comment]
@kevGalway thank you for your review. Pipelines aren't editable via API, so I didn't add any word about it. I've made some changes according to your comment, so please take a look at it

**Comment by kevGalway** [comment]
Looks good.

**Pull request merged**